documentation/modules/auxiliary/scanner/smb/smb_login.md

```diff
@@ -37,8 +37,158 @@ host          origin        service        public  private  realm  private_type
+   DETECT_ANY_AUTH    false            no        Enable detection of systems accepting any authentication
+   DETECT_ANY_DOMAIN  false            no        Detect if domain is required for the specified user
+   PASS_FILE                           no        File containing passwords, one per line
+   PRESERVE_DOMAINS   true             no        Respect a username that contains a domain name.
+   Proxies                             no        A proxy chain of format type:host:port[,type:host:port][...]
+   RECORD_GUEST       false            no        Record guest-privileged random logins to the database
+   RHOSTS             172.16.158.154   yes       The target host(s), see https://docs.metasploit.com/docs/using-metasploit/basics/using-metasploit.html
+   RPORT              445              yes       The SMB service port (TCP)
+   SMBDomain          windomain.local  no        The Windows domain to use for authentication
+   SMBPass            vagrant          no        The password for the specified username
+   SMBUser            vagrant          no        The username to authenticate as
+   STOP_ON_SUCCESS    false            yes       Stop guessing when a credential works for a host
+   THREADS            1                yes       The number of concurrent threads (max one per host)
+   USERPASS_FILE                       no        File containing users and passwords separated by space, one pair per line
+   USER_AS_PASS       false            no        Try the username as the password for all users
+   USER_FILE                           no        File containing usernames, one per line
+   VERBOSE            true             yes       Whether to print output for all attempts
+
+
+View the full module info with the info, or info -d command.
+
+msf6 auxiliary(scanner/smb/smb_login) > run
+```
+
+Should give you output containing
+
+```msf6
+[*] 172.16.158.154:445    - 172.16.158.154:445 - Starting SMB login bruteforce
+[+] 172.16.158.154:445    - 172.16.158.154:445 - Success: 'windomain.local\vagrant:vagrant' Administrator
+[*] SMB session 1 opened (172.16.158.1:62793 -> 172.16.158.154:445) at 2024-03-12 17:03:09 +0000
+[*] 172.16.158.154:445    - Scanned 1 of 1 hosts (100% complete)
+[*] Auxiliary module execution completed
+msf6 auxiliary(scanner/smb/smb_login) > sessions -1
+[*] Starting interaction with 1...
+```
+
+Which you can interact with using `sessions -i <session id>` or `sessions -1` to interact with the most recently opened session.
+
+```msf6
+msf6 auxiliary(scanner/smb/smb_login) > sessions -1
+[*] Starting interaction with 1...
+
+SMB (172.16.158.154) > shares
+Shares
+======
+
+    #  Name    Type          comment
+    -  ----    ----          -------
+    0  ADMIN$  DISK|SPECIAL  Remote Admin
+    1  C$      DISK|SPECIAL  Default share
+    2  foo     DISK
+    3  IPC$    IPC|SPECIAL   Remote IPC
+
+SMB (172.16.158.154) > shares -i foo
+[+] Successfully connected to foo
+SMB (172.16.158.154\foo) > ls
+ls
+===
+...
+```
```

Comment:
is the full output of ls intended to be here? can't tell if it's been lost or purposely truncated

Reply:
I purposely truncated it since it seemed unnecessary - any suggestions for making that less confusing?

Reply:
it's probably fine as is, we don't have a super consistent way of labelling something as truncated, maybe `[truncated]` would be good? I'm not super concerned as long as it was intentional though